addons/purchase_stock/views/stock_views.xml

```diff
@@ -37,4 +37,15 @@
             </field>
         </record>
 
+        <record id="warehouse_orderpoint_search_inherited_purchase_stock" model="ir.ui.view">
```

Comment:
It would be great to start following guidelines for xml https://www.odoo.com/documentation/16.0/fr/contributing/development/coding_guidelines.html#xml-files